### DISCUSSION TITLE
Quick way to translate the value attribute

### DISCUSSION THREAD

**Discussion by samuel-clara** [solspace/craft-freeform#1443] [Questions & Answers] [answered]
Is there a quick way to apply |t('freeform') to the option "value" attribute of all dropdown fields?

I have to build a pretty complex form with a lot of option and it would be super useful if I could use OPTION_1, OPTION_2 ecc. strings and then translate these with freeform.php. This actually works except for the fields values which are obviously also visible in the "submission" section and in the notification emails.

**Comment by kjmartens** [comment]
Hi @samuel-clara,

This should automatically happen when using `{{ field.render }}` or `{{ field.renderInput }}`, etc. If you're doing things manually, you could also do that like this:

``` twig
<label>My Dropdown Field</label>
<select name="{{ myDropdown.handle }}">
{% for option in myDropdown.options %}
    <option value="{{ option.value }}" {{ option.value in field.value ? "selected" }}>{{ option.label|t('freeform') }}</option>
{% endfor %}
</select>
```

If you're referring to something different, can you share what your template code looks like and elaborate? 🙂

**Reply by samuel-clara** [reply]
I'm referring to the value attribute of the options of a dropdown element/select html tag. Those can't be translated?

**Comment by kjmartens** [comment, marked as answer]
Unfortunately the actual option value cannot be translated for option-based fields like Dropdowns. This is because Freeform expects values that are pre-configured in the form builder.

You have two options to workaround this issue:

1. Create a duplicate of the form and update that to have different values (e.g., one form in English, the other in French).
2. You could add the dropdown field as a regular TEXT field type (then override it in your formatting template to display as a dropdown). With this approach, the value could be anything, and if it's translated in the form during submission, it should be stored as the translated value as well, meaning that it would display in the Submissions area and email notifications as the translated value you are expecting. 🙂

**Reply by samuel-clara** [reply]
Option one is definitely too difficult to maintain but I will explore option 2. Thank you 😊
